### DISCUSSION TITLE
【网址参数清理】可以直接清理页面a[href]中的链接参数

### DISCUSSION THREAD

**Discussion by MkQtS** [the1812/Bilibili-Evolved#4543] [功能建议 / Ideas]
例如从原来的这样：

![image](https://github.com/the1812/Bilibili-Evolved/assets/81752398/8eb23c84-7d25-4385-ab5f-ebcc35cb6791)

改成这样：

![image](https://github.com/the1812/Bilibili-Evolved/assets/81752398/56618cc9-4e0f-4b53-b20c-8a8f3de9afe6)

在用户点击链接前，清理了`spm_id_from`，同时防止能追踪到用户ID的`vd_source`参数生成，也不会影响历史记录

**Comment by the1812** [comment]
1. 不是所有 b 站的链接都是 `a[href]`
2. 点击的瞬间 b 站还会给你自动加一个 `spm_id_from`, 见 #2247
